### ISSUE TITLE
[Avatar]: decorative property should be supported 

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents-react#6352]
### Is your feature request related to a problem?

Yes. In some cases Avatar is just used for decorative purpose and should be ignored by accessibility tools. 
For example: Avatar user name is shown in text beside the avatar image, In this case Screen reader will read user name from text and also from Avatar accessibleName as shown below
![Screenshot 2024-09-13 at 4 42 33 PM](https://github.com/user-attachments/assets/598c6207-fbfa-4e4f-8f3b-326cc1fa5ae4)

### Describe the solution you'd like

Provide decorative property like : https://ui5.sap.com//#/api/sap.m.Avatar%23controlProperties

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
